problems/isOdd.js

```diff
@@ -7,8 +7,12 @@
 
 */
 
-function isOdd() {
-
+function isOdd(n) {
+    if(n % 2 === 1){
+        return true;
+    }else{
+        return false
+    }
```

Comment:
You can also write this in one line by just returning the boolean result of the module comparison